samples/apim-policy/request-routing-policy.xml

```diff
@@ -0,0 +1,50 @@
+<!--
```

Comment:
I'm assuming someone that has knowledge of APIM (ideally the person reading this sample) would know where to put this xml and how to use that. Which is to say, I'm not sure how to do it but if you agree it's common knowledge for a user of apim, then not worth expanding on usage.

Reply:
I believe this would be common knowledge for an APIM user since policies are an integral part of the API definition process.